xbmc/ServiceBroker.h

```diff
@@ -149,6 +154,7 @@ class CServiceBroker
   static WSDiscovery::IWSDiscovery& GetWSDiscovery();
   static MEDIA_DETECT::CDetectDVDMedia& GetDetectDVDMedia();
   static PVR::CPVRManager& GetPVRManager();
+  static KODI::GUILIB::CGUITextureJobManager& GetTextureJobManager();
```

Comment:
Does this need to be a new "service" accessible from the ServiceBroker - will this be used by anything other than the GUILargeTextureManager?
LargeTextureManager has a fairly long lifetime, if that is the only thing that needs to upload texture to GPU this could be a private member of that.

Reply:
Yes, I'd like to also use it for a dedicated background layer, which is not part of the GUI. On this layer, we could display fanart or photos at higher resolution, while still keeping the GUI responsive.